requirements.txt

```diff
@@ -15,6 +15,7 @@ psycopg2-binary>=2.7.4
 python-memcached==1.58
 raphendyr-django-essentials==1.3.0
 ansi2html==1.2.0
+plotly==4.5.0
```

Comment:
Should use relative requirement `plotly ~= 4.5.0`, so point releases are used instead.